test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -49,8 +66,8 @@ public string[] GetInstallCommandsFromLocal(string path)
                 SupportedPackageExtension.Rpm => new[]
                 {
                     "set -e",
-                    $"yum install -y {string.Join(' ', packages)}",
-                    "pathToSystemdConfig=$(systemctl cat iotedge | head -n 1)",
+                    $"rpm --nodeps -i {string.Join(' ', packages)}",
```

Comment:
Nit question: why switch this?

Reply:
The version numbers between the aziot-identity-service and aziot-edge packages were incompatible:
```
Error: Package: aziot-edge-1.1.0-0.1.dev20201123.5.el7.x86_64 (/aziot-edge-1.1.0-0.1.dev20201123.5.el7.x86_64)
           Requires: aziot-identity-service >= 1.1.0-0.1.dev20201123.5
```

`yum` was refusing to install aziot-edge without an appropriately numbered aziot-identity-service, so I changed it to `rpm --nodeps` which ignores the dependencies.